app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,12 @@
 
     <%= stylesheet_link_tag    'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
+    <!-- Currently placing these links in the header -->
+    <a href="/movies">Movies</a>
+    <a href="/directors">Directors</a>
+    <!-- <a href="/discs">Discs</a> -->
+    <!-- <a href="/producers">Producers</a> -->
+    <!-- we can move later if a User Story denotes a specific location -->
```

Comment:
This makes sense, and this layout/application will show up on every page?

Reply:
Yes --- anything we put in the application.html.erb file will apply to all pages. I put in a placeholder for your pages as well to illustrate how this could work.